### PR TITLE
Docs Cleanup in preparation of new build process

### PR DESCRIPTION
**Pull request by gamarin2**
Title

- Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/master/CONTRIBUTING.md#pr-targeting))

- [ ] Linked to github-issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests
- [ ] Updated relevant documentation (`docs/`)
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Reviewed `Files changed` in the github PR explorer

______

For Admin Use:
- Added appropriate labels to PR (ex. wip, ready-for-review, docs)
- Reviewers Assigned
- Squashed all commits, uses message "Merge PR #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,22 +9,13 @@
 [![GolangCI](https://golangci.com/badges/github.com/cosmos/gaia.svg)](https://golangci.com/r/github.com/cosmos/gaia)
 [![riot.im](https://img.shields.io/badge/riot.im-JOIN%20CHAT-green.svg)](https://riot.im/app/#/room/#cosmos-sdk:matrix.org)
 
-This repository hosts `Gaia`, the first implementation of the Cosmos Hub.
+This repository hosts `Gaia`, the first implementation of the Cosmos Hub based on the [Cosmos SDK](https://github.com/cosmos/cosmos-sdk).
 
 **Note**: Requires [Go 1.13+](https://golang.org/dl/)
 
-**DISCLAIMER**: The current version of Gaia running the Cosmos Hub (v0.34.x) is
```
Comment (alexanderbez) [inline README.md:16]: Why was this removed? I added it because we kept getting github issues created stating their binary wasn't working and that was because they were using this repo!

**Review by alexanderbez — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (gamarin2) [reply README.md:16]: Right @alexanderbez, I thought I opened it as a draft, my bad. It should not be merged before the hub upgrade

Comment (alexanderbez) [reply README.md:16]: got it
